### ISSUE TITLE
[Bug]: JSONDecodeError

### ISSUE THREAD

**Issue by EdyDeveloper** [ThioJoe/YT-Spammer-Purge#479]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I put the client_secrets.json file in and then tried running the program but it says this:

```
Traceback (most recent call last):
  File "Scripts\auth.py", line 81, in first_authentication
  File "Scripts\auth.py", line 68, in get_authenticated_service
  File "google_auth_oauthlib\flow.py", line 202, in from_client_secrets_file
  File "json\__init__.py", line 293, in load
  File "json\__init__.py", line 346, in loads
  File "json\decoder.py", line 340, in decode
json.decoder.JSONDecodeError: Extra data: line 1 column 13 (char 12)
----------------
[!!!] Error: Extra data: line 1 column 13 (char 12)
If you think this is a bug, you may report it on this project's GitHub page: https://github.com/ThioJoe/YT-Spammer-Purge/issues

Error Code A-1: Something went wrong during authentication. Try deleting the token.pickle file.
Press Enter to exit...
```

![image](https://user-images.githubusercontent.com/63300247/152115125-1c456722-5d37-4db0-96a9-7c1706e31557.png)

### Release version

2.14.2

### Steps to reproduce

1. Dowonload the exe
2. Put the API key in the client_secrets.json file in the same directory as the exe
3. Run the exe

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Loading YT Spammer Purge @ 2.14.2...

Please login using the browser window that opened just now.



Traceback (most recent call last):
  File "Scripts\auth.py", line 81, in first_authentication
  File "Scripts\auth.py", line 68, in get_authenticated_service
  File "google_auth_oauthlib\flow.py", line 202, in from_client_secrets_file
  File "json\__init__.py", line 293, in load
  File "json\__init__.py", line 346, in loads
  File "json\decoder.py", line 340, in decode
json.decoder.JSONDecodeError: Extra data: line 1 column 13 (char 12)
----------------
[!!!] Error: Extra data: line 1 column 13 (char 12)
If you think this is a bug, you may report it on this project's GitHub page: https://github.com/ThioJoe/YT-Spammer-Purge/issues

Error Code A-1: Something went wrong during authentication. Try deleting the token.pickle file.
Press Enter to exit...
```

### Screenshots

![image](https://user-images.githubusercontent.com/63300247/152115686-b96eb5d8-89f0-478f-872d-96c95a86e6d5.png)

**Comment by Dobby233Liu** [comment]
Did you put your secrets file in OneDrive, I heard there's some problem about file contents in it

**Comment by Dobby233Liu** [comment]
Also see: https://github.com/ThioJoe/YT-Spammer-Purge/issues/453

**Comment by EdyDeveloper** [comment]
Thanks for pointing me to that.
When I tried searching for something similar it did not come up.
I tried both one my desktop (that's why it was onedrive) and on a separate disk.
![image](https://user-images.githubusercontent.com/63300247/152123840-72bc2db1-1185-416c-a36a-7452e5533ddf.png)

**Comment by KendallDoesCoding** [comment]
> Thanks for pointing me to that. When I tried searching for something similar it did not come up. I tried both one my desktop (that's why it was onedrive) and on a separate disk. ![image](https://user-images.githubusercontent.com/63300247/152123840-72bc2db1-1185-416c-a36a-7452e5533ddf.png)

If the issue is fixed, requesting you to close this issue thread. If not, pleas update the title accordingly

**Comment by ThioJoe** [comment]
Likely duplicate of #453 . I assume you copy and pasted the contents into the client_secrets.json file, instead of actually downloading the file?

**Comment by ThioJoe** [comment]
Ok this is now addressed in release 2.14.3 and https://github.com/ThioJoe/YT-Spammer-Purge/commit/e16ebb9d520e65f12952cfc27c69272233b8bd65

It tells the user they made the client secrets json wrong. I'll re-open if OP comes back and says it was something else
